ansible_base/authentication/constants.py

```diff
@@ -0,0 +1,4 @@
+TEAM_MEMBER_ROLE_NAME = "Team Member"
+TEAM_ADMIN_ROLE_NAME = "Team Admin"
+ORGANIZATION_MEMBER_ROLE_NAME = "Organization Member"
+ORGANIZATION_ADMIN_ROLE_NAME = "Organization Admin"
```

Comment:
We might want to make these properties in the `ReconcileUser` class so that if a specific service had a different role name for some reason they could extend their `ReconcileUser` class to change the property.